src/main/java/de/retest/recheck/ui/image/ImageUtils.java

```diff
@@ -194,6 +194,9 @@ public static BufferedImage readImage( final String path ) throws IOException {
 
 	public static Image scaleProportionallyToMaxWidthHeight( final BufferedImage image, final int maxWidth,
 			final int maxHeight ) {
+		if ( image == null ) {
```

Comment:
You really need to [look for the cause, not fix the symptom](https://www.secureworks.com/blog/treat-the-cause-not-the-symptom).